src/screens/Login.js

```diff
@@ -79,37 +108,24 @@ class Login extends Component {
 
   onEmailChange = (email) => {
     const { user: { password } } = this.state;
-    this.setState({ user: { email, password } });
+    this.setState({ user: { email, password }, error: '' });
   };
 
   onPasswordChange = (password) => {
```

Comment:
These kind of functions are quite similar. I mean `onPasswordChange` and `onEmailChange`. Maybe this could be one function with an extra paramter and an arrow function in component props?